python/pysrc/kaskada/windows.py

```diff
@@ -16,10 +16,15 @@ class Window(object):
 
 @dataclass(frozen=True)
 class Since(Window):
-    """Window since the last time a predicate was true.
+    """A series of non-overlapping windows that reset each time a predicate evaluates to true.
 
     Aggregations will contain all values starting from the last time the predicate
     evaluated to true (inclusive).
```

Comment:
This sounds incorrect. Shouldn't it be *exclusive* of the last time the predicate was true?

Reply:
Yeah, I thought it sounded weird too. But it's more so "If an `input` exists in the same row that the `predicate` is true, it will be added to the previous window". 

So maybe it should be simply reworded to "Aggregations are exclusive on start and inclusive on end". Then the next sentence should clarify whether it emits for all inputs or on window boundaries.